### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ADHIRAJ-12**
Updated commands for cloning and running the project.

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
thanks for your contribution @ADHIRAJ-12 : I added some small comments

**Review by pascalgrimaud — CHANGES_REQUESTED** [review]

**Comment by pascalgrimaud** [comment]
Oh sorry, I forgot to publish my comments

File: README.md [hunk 1]
```diff
@@ -33,14 +33,16 @@ This is a [sample application](https://github.com/jhipster/jhipster-lite-sample-
 
 ## Quick Start
 
-You need to clone this project and go into the folder:
-
+Open your code editor and type the following command  to clone:
```
[the remark this answers is not in the document]
Comment (ADHIRAJ-12) [reply README.md:36]: oh that's absolutely correct !

**Review by ADHIRAJ-12 — COMMENTED** [review]
made the changes as suggested

**Comment by pascalgrimaud** [comment]
Still not good. The missing space is here, between each blocks:

image

**Review by ADHIRAJ-12 — COMMENTED** [review]
added spaces according to the original code

**Review by ADHIRAJ-12 — COMMENTED** [review]
done sir

File: README.md [hunk 2]
```diff
@@ -33,21 +33,19 @@ This is a [sample application](https://github.com/jhipster/jhipster-lite-sample-
 
 ## Quick Start
 
-You need to clone this project and go into the folder:
-
+Open your terminal and type the following command to clone:
 ```
 git clone https://github.com/jhipster/jhipster-lite
+```
+To go into the folder type:
```
Comment (pascalgrimaud) [inline README.md:40]:
```suggestion

To go into the folder type:

```

File: README.md [hunk 3]
```diff
@@ -33,21 +33,19 @@ This is a [sample application](https://github.com/jhipster/jhipster-lite-sample-
...
 cd jhipster-lite
 ```
-
 Run the project:
-
-```bash
-./mvnw
 ```
-
+bash./mvnw
```
Comment (pascalgrimaud) [inline README.md:46]: The syntax is still wrong, and this part should be unchanged

**Comment by pascalgrimaud** [comment]
the CI failed, need to run prettier

**Pull request closed without merging**